src/tests/fs_tests.rs

```diff
@@ -786,7 +889,7 @@ pub mod fs_tests {
 
         //now we are going to mknod /dev/null with create, read and write flags and permissions
```

Comment:
Shouldn't the test cases include mknod on different combinations of read/write/create permissions?

Reply:
Added the cases for different combinations. Actually, the `S_IRWXA` flag is the best possible combination for all the permissions, which covers all cases.